wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -82,9 +83,8 @@ public void compile() throws IOException {
       schema = schema.retainRoots(options.roots);
     }
 
-    JavaGenerator javaGenerator = JavaGenerator.get(schema);
-    TypeWriter typeWriter = new TypeWriter(
-        javaGenerator, options.emitOptions, options.enumOptions);
+    JavaGenerator javaGenerator = JavaGenerator.get(schema)
+        .withOptions(options.emitOptions, options.enumOptions);
```

Comment:
(I think this is a really cute way to make it easy to get one of these without thinking about options, while still making options available to those of us who need 'em)